app/controllers/admin/podcasts_controller.rb

```diff
@@ -34,21 +35,12 @@ def fetch
       redirect_to admin_podcasts_path
     end
 
-    def remove_admin
-      removed_roles = @user.remove_role(:podcast_admin, @podcast)
-      if removed_roles.empty?
-        redirect_to edit_admin_podcast_path(@podcast), notice: "Error"
-      else
-        redirect_to admin_podcasts_path, notice: "Removed admin"
-      end
-    end
-
-    def add_admin
-      role = @user.add_role(:podcast_admin, @podcast)
-      if role.persisted?
-        redirect_to admin_podcasts_path, notice: "Added admin"
+    def add_owner
+      @podcast_ownership = @podcast.podcast_ownerships.build(user_id: params["podcast"]["user_id"])
+      if @podcast_ownership.save
+        redirect_to admin_podcasts_path, notice: "New owner added!"
       else
-        redirect_to edit_admin_podcast_path(@podcast), notice: "Error"
+        redirect_to edit_admin_podcast_path(@podcast), notice: @podcast_ownership.errors_as_sentence
```

Comment:
Great use of `.errors_as_sentence` here! 😄

Reply:
@rhymes thanks for suggesting that! :smiley: